### PR TITLE
Improved localization contribution guide for Kubernetes

### PR DESCRIPTION
**Pull request by kumarankit999**
I have  break up the Localization Contribution guide into 3 pages that focus on

1. Contributing to a localization
2. Starting a new localization team
3. Maintaining a localization
which resolves the issue #43116

### CODE REVIEW

File: content/en/docs/contribute/Localization/Start_a_localization.md [hunk 1]
```diff
@@ -1,560 +1,338 @@
...
-## Upstream contributions
-
-SIG Docs welcomes upstream contributions and corrections to the English source.
+---
+title: Starting an Existing Localization
+content_type: concept
+approvers:
+- remyleone
+- rlenferink
+weight: 50
```
[the remark this answers is not in the document]
Comment (kumarankit999) [reply content/en/docs/contribute/Localization/Start_a_localization.md:7]: How Much????

File: content/en/docs/contribute/Localization/Contribute_in_localization.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+For extra details on how to contribute to a specific localization,
+look for a localized version of this page.
+{{< /note >}}
+
+
+<!-- body -->
+
+### Find your two-letter language code
+
+First, consult the
```
Comment (Ritikaa96) [inline content/en/docs/contribute/Localization/Contribute_in_localization.md:35]:
```suggestion
You can consult the

```

Comment (Ritikaa96) [reply content/en/docs/contribute/Localization/Contribute_in_localization.md:35]: As there is no second point in this section, rewording it to statement will be better.

**Comment by reylejano** [comment]
Hi @kumarankit999 , since this PR needs rebase to fix a merge conflict, I'm closing it.
When you have time to work on this PR and rebase, you can reopen this PR with a `/reopen` in a comment
/close

**Pull request closed without merging**